styles/styles.css

```diff
@@ -117,3 +152,185 @@ ul li a img {
   color: var(--social-color);
   margin-right: 1.25rem;
 }
+
+/* Work section */
+#works {
+  padding: 7.125rem 1.5rem;
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.card {
+  margin-bottom: 5.5rem;
+  padding: 1rem 1rem 0 1rem;
+  background-color: white;
+  border: 1px solid var(--card-border-color);
+  border-radius: 1rem;
+}
+
+.card img {
```

Comment:
- [ ] Please make sure that your image in your card is centered like on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=sG31kPTSmeBIdEXV-0)

Your design 👇👇👇

![Screenshot from 2023-06-02 02-01-13](https://github.com/jecode93/portfolio/assets/10439283/747763ba-5b22-48e9-babd-fac2aa94f431)

Design from Figma 👇👇👇

![Screenshot from 2023-06-02 02-00-13](https://github.com/jecode93/portfolio/assets/10439283/a080e0e6-eeaa-4e74-ae5a-6037654bc273)

Reply:
Hello @rica213,

Thanks a lot for your review! I really like the way you answer...
But it appears to be good on my part, so I will try to see what going wrong as you said.  And for the font family, I use the same as used in the Figma template

<img width="402" alt="Screen Shot 2023-06-02 at 3 51 30 AM" src="https://github.com/jecode93/portfolio/assets/45808742/54d01d47-df36-4401-b63b-255f55ed6a50">

Reply:
@rica213 Sorry if I say that, but maybe the way you saw the image before is due to your screen size, I did my design for a 375 px screen as requested in the project description... The screenshot above is from my design it appears to be exactly the same as the Figma template!!
Please, consider seeing the web page on a 375px wide and tell me if it is the same 🙏🏼 🙏🏼 🙏🏼

Reply:
Hi Jean Emmanuel, kindly check the videoscreen below 👇👇 This is what I meant 🥲
It is supposed to be centered for mobile device, not just the width 375px. I hope that answers your question ❓❓❓

[Screencast from 2023-06-02 11-14-20.webm](https://github.com/jecode93/portfolio/assets/10439283/fcc52137-7279-4a21-9bc6-9364dc11233d)

Reply:
@rica213 yes I understand, you are right... I will do the changes now!

Reply:
@rica213 I think everything should be ok now. 
I finish with the changes.

Reply:
@rica213 So everything is ok now, can I request another review for approval?